apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx

```diff
@@ -334,51 +345,81 @@ export const BrandingPreferencePreview: FunctionComponent<BrandingPreferencePrev
-                                            title={
-                                                layoutContext[0] === PredefinedLayouts.CUSTOM
-                                                    ? t("extensions:develop.branding.tabs.preview.errors."
-                                                        + "layout.notFoundWithSupport.title")
-                                                    : t("extensions:develop.branding.tabs.preview.errors."
-                                                        + "layout.notFound.title")
+                                                            { t("common:learnMore") }
+                                                        </DocumentationLink>
+                                                    </>
+                                                ]
                                             }
+                                            title={ t("extensions:develop.branding.tabs.preview."
+                                                + "info.layout.activatedMessage.title") }
                                         />
                                     </div>
+                                ) : (
+                                    isErrorOccured
+                                        ? (
+                                            <div className="branding-preference-preview-message" >
+                                                <EmptyPlaceholder
+                                                    image={ CustomLayoutWarningImg }
+                                                    imageSize="small"
+                                                    subtitle={
+                                                        layoutContext[0] === PredefinedLayouts.CUSTOM
+                                                            ? [
+                                                                t("extensions:develop.branding.tabs.preview."
+                                                                    + "errors.layout.notFoundWithSupport.subTitle"),
+                                                                <Trans
+                                                                    key={ 1 }
+                                                                    i18nKey={ "extensions:develop.branding."
+                                                                        + "tabs.preview.errors.layout."
+                                                                        + "notFoundWithSupport.description" }
+                                                                    tOptions={ {
+                                                                        supportEmail
+                                                                    } }
+                                                                >
+                                                                    Need a fully customized layout for your 
```

Comment:
Don't we need to i18n these strings?

Reply:
Already added the i18n. This is the `Trans` component (You can see the provided `i18nKey` as a prop) and the code is an existing one.